### PR TITLE
[MRG+1]: Add source links, fix CSS

### PR DESCRIPTION
**Pull request by Eric89GXL**
This:

1. ~~Uses more standard bootstrap CSS styles instead of custom ones~~
2. ~~Changes flatly->cerulean since it matches the markup a bit better (can be reverted if people hate it)~~
3. Adds "[source]" links to our API pages
4. Fixes the notebook links at bottom of examples
5. Does minor reorganization / de-duplication of items.
6. Adds minor tweaks to a couple of examples.
7. Gives our logo a transparent background (useful if we embed it in a Jumbotron sometime)

Ready for review/merge from my end. Will post a link to the build when it's done.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Oh my precious GitHub, so glad you're back. (Stupid DDoS going on made it inaccessible to me for most of the day.) Rendered HTML:

https://2852-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/index.html

File: doc/faq.rst [hunk 1]
```diff
@@ -87,6 +113,26 @@ which is based on the
 h5py_, to save data in a fast, future-compatible, standard format.
 
 
+I downloaded a dataset once, but MNE-Python is asking to download it again. What gives?
```
Comment (jasmainak) [inline doc/faq.rst]: "What gives?" ?

**Comment by jasmainak** [comment]
Source code links are cool @Eric89GXL !

File: doc/cite.rst [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-    [1] A. Gramfort, M. Luessi, E. Larson, D. Engemann, D. Strohmeier, C. Brodbeck, L. Parkkonen, M. Hämäläinen, `MNE software for processing MEG and EEG data <http://www.ncbi.nlm.nih.gov/pubmed/24161808>`_, NeuroImage, Volume 86, 1 February 2014, Pages 446-460, ISSN 1053-8119, `[DOI] <http://dx.doi.org/10.1016/j.neuroimage.2013.10.027>`__
-
-If you use the Python code you should cite as well:
-
-    [2] A. Gramfort, M. Luessi, E. Larson, D. Engemann, D. Strohmeier, C. Brodbeck, R. Goj, M. Jas, T. Brooks, L. Parkkonen, M. Hämäläinen, `MEG and EEG data analysis with MNE-Python <http://journal.frontiersin.org/article/10.3389/fnins.2013.00267/abstract>`_, Frontiers in Neuroscience, Volume 7, 2013, ISSN 1662-453X, `[DOI] <http://dx.doi.org/10.3389/fnins.2013.00267>`__
-
-To cite specific versions of the software, you can use the DOIs provided by
-`Zenodo <https://zenodo.org/search?ln=en&p=mne-python>`_.
-
-You should as well cite the related method papers, some of which are listed in :ref:`ch_reading`.
```
Comment (agramfort) [inline doc/cite.rst]: well I think it's easier to find if you keep it in it's own page so we can access it from the font page not just in a subsection of FAQ.

**Comment by agramfort** [comment]
looks nice to me. Just one nitpick: there is more white space on front page around MNE logo with this new design.

is this fixable?

File: doc/cite.rst (hunk 2 above)
Comment (Eric89GXL) [inline doc/cite.rst]: If the issue is having a one-click link, I could just link to the page in the FAQ. Would that be okay? It seems overkill to give it its own page for so little information.

**Comment by Eric89GXL** [comment]
> is this fixable?

It is *changeable*, whether you think such a modification is a *fix* or *removal of an enhancement* is an aesthetic judgment :) But sure, I can reduce it

**Comment by agramfort** [comment]
please try to reduce it. thanks

**Comment by Eric89GXL** [comment]
@agramfort done

https://2871-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/index.html

**Comment by agramfort** [comment]
@dengemann @jasmainak @jaeilepp et al. you're all happy?

**Comment by jaeilepp** [comment]
https://2852-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/tutorials.html looks a bit messed up.

**Comment by jasmainak** [comment]
The website looks a bit too blue to me. I am not good with picking colors myself, so can't give any good suggestions ... I'll defer to the majority opinion :)

**Comment by Eric89GXL** [comment]
@jaeilepp what is messed up? I think it's okay if not all boxes are the same size. I also left one out of the upper right to keep similar things together later (like the source/sensor stats, for example).

**Comment by jaeilepp** [comment]
The boxes are not very well aligned 
screenshot_2016-10-25_15-24-49

**Comment by jaeilepp** [comment]
I liked the old one better.

**Comment by Eric89GXL** [comment]
Most style changes reverted:

https://2883-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/index.html

**Comment by agramfort** [comment]
+1 for merge

**Comment by agramfort** [comment]
looks good

while browsing I saw this pb of line break on citation at:

https://2871-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/connectivity/plot_cwt_sensor_connectivity.html#compute-seed-based-time-frequency-connectivity-in-sensor-space

**Comment by Eric89GXL** [comment]
Must be an American idiom. Like "what's going on". I'll change it

**Comment by Eric89GXL** [comment]
@agramfort done

https://2889-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/connectivity/plot_cwt_sensor_connectivity.html#sphx-glr-auto-examples-connectivity-plot-cwt-sensor-connectivity-py

**Comment by jasmainak** [comment]
yeah looks good to me to!

**Comment by agramfort** [comment]
Great
Merge if you are happy

**Comment by jasmainak** [comment]
Thanks @Eric89GXL !

**Pull request merged**